index.js

```diff
@@ -1 +1,115 @@
-// Do work!
\ No newline at end of file
+function validatePassword(password) {
+  const secureLength = (password.length >= 8) ? true : false
+
+
```

Comment:
You can see that jquery is still more elegant.

This can likely be shortened up a bit but still requires more code than jquery:
document.getElementById("length") is a native option.
document.getElementById("length").classList.remove("valid");
 document.getElementById("length").classList.add("invalid");